src/vendor/d3v5.js

```diff
@@ -1,11 +1,11 @@
-// https://d3js.org v5.7.0 Copyright 2018 Mike Bostock
+// https://d3js.org v5.16.0 Copyright 2020 Mike Bostock
```

Comment:
I'm a bit concerned about the upgrade of this library. I'd be good to make a list of Banana panels/modules that depend on d3v5 so we know which ones could have issues if v5.7.0 and v5.16.0 are not entirely compatible.

Reply:
This file seems the only one that uses the d3v5; I added a Note at the beginning of the file:

    
    Note: 
        this is the only file that uses the version 5 of the d3 library (d3v5).
        The d3v5 library has been updated from v5.7.0 up to v5.16.0.